README.md

```diff
@@ -41,6 +41,7 @@ This repository will have collection of such aliases. Read [Contribution Guideli
     - [Get saved WiFi keys](#get-saved-wifi-keys)
     - [Take Screenshot of connected ADB Device](#take-screenshot-of-connected-adb-device)
     - [Bootstrap your CF Round](#bootstrap-your-cf-round)
+    - [Run Matlab scripts](#run-matlab-scripts)
```

Comment:
The id you called is `matlab` so make it consistent, else clicking won't redirect.

Reply:
Done!